cluster_1: catch( AssertionError e ) {
cluster_1: catch( final AssertionError e ) {
cluster_1: catch( AssertionError e ) { error = e ;
cluster_1: catch( AssertionError e ) { error =
cluster_1: catch( AssertionError e ) { error = null ;
cluster_1: catch( AssertionError e ) { error = error ;
